src/lib/propertiesConverters.js

```diff
@@ -9,26 +9,15 @@
 import _ from 'lodash'
 import { format } from 'd3-format'
 
-export const getLabelGenerator = (_label, labelFormat) => {
-    if (_.isFunction(_label)) {
-        return _label
-    }
-
-    const label = d => _.get(d, _label)
+export const getLabelGenerator = (_label, labelFormat = '') => {
+    if (_.isFunction(_label)) return _label
 
-    let formatter
-    if (labelFormat) {
-        formatter = format(labelFormat)
-    }
+    const formatter = format(labelFormat)
```

Comment:
We should keep the condition and only apply it if it's necessary, I think it can make a difference when we have large graphs